### PR TITLE
✨ Introduce env-vars to control replication of catalog, director-v0, dask-scheduler

### PR DESCRIPTION
**Pull request by mrnicegyu11**
## What do these changes do?
- Introduce env-vars to control replication of catalog, director-v0, dask-scheduler

Bonus:
- 🐛 Fixes grafana provisioning on awsmaster

## Related issue/s
https://github.com/ITISFoundation/osparc-ops-environments/issues/461
## Related PR/s
https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/merge_requests/221
## Checklist

- [x] I tested and it works

### CODE REVIEW

**Comment by YuryHrytsuk** [comment]
Did we already decide to use AWS MQ for all AWS deployments?

**Comment by mrnicegyu11** [comment]
> Did we already decide to use AWS MQ for all AWS deployments?

no, good catch this is an error in the PR i didnt want to commit those lines. Thanks!

**Review by sanderegg — APPROVED** [review]
:+1:

File: services/simcore/docker-compose.deploy.local.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ services:
       - SSL_CERT_FILE=/usr/local/share/ca-certificates/osparc.crt
   director:
     deploy:
-      replicas: 1
+      replicas: ${SIMCORE_DIRECTOR_V0_REPLICAS}
     environment:
       # needed to pass the self-signed certificate to the spawned services
```
Comment (YuryHrytsuk) [inline services/simcore/docker-compose.deploy.local.yml:26]: Forgot `SIMCORE_DASK_SCHEDULER_REPLICAS` for local compose?

**Review by YuryHrytsuk — APPROVED** [review]

File: services/simcore/docker-compose.deploy.local.yml (hunk 1 above)
Comment (mrnicegyu11) [reply services/simcore/docker-compose.deploy.local.yml:26]: lgtm as discussed personally :--) much appreciated

**Pull request merged**